kornia/augmentation/auto/base.py

```diff
@@ -14,12 +15,21 @@
 SUBPLOLICY_CONFIG = List[OP_CONFIG]
 
 
-class PolicyAugmentBase(ImageSequentialBase):
+class PolicyAugmentBase(TransformMatrixMinIn, ImageSequentialBase):
     """Policy-based image augmentation."""
 
-    def __init__(self, policy: List[SUBPLOLICY_CONFIG]) -> None:
+    def __init__(self, policy: List[SUBPLOLICY_CONFIG], transformation_matrix_mode: str = "silence") -> None:
         policies = self.compose_policy(policy)
         super().__init__(*policies)
+        self._parse_transformation_matrix_mode(transformation_matrix_mode)
+        self._valid_ops_for_transform_computation: Tuple[Any, ...] = (PolicySequential,)
+
+    def _update_transform_matrix_for_valid_op(self, module: PolicySequential) -> None:  # type: ignore
```

Comment:
what is the issue type here?

Reply:
IIRC, module shall be a `nn.Module` than `PolicySequential`